src/lib/tls/tls_channel.cpp

```diff
@@ -168,6 +180,15 @@ Handshake_State& Channel::create_handshake_state(Protocol_Version version)
          m_sequence_numbers.reset(new Stream_Sequence_Numbers);
       }
 
+   for (uint64_t seq = 0; seq < pre_in_message_seq; seq++)
+      {
+      m_sequence_numbers->read_accept(seq);
+      }
+   for (uint16_t seq = 0; seq < pre_out_message_seq; seq++)
+      {
+      m_sequence_numbers->next_write_sequence(0);
```

Comment:
Use `/*epoch=*/0` to make it clear what this is - at first I thought this was a typo and should have been `seq`